src/liveEditor/utils/handleIndividualFields.ts

```diff
@@ -137,7 +141,6 @@ export async function handleIndividualFields(
                 }
             );
 
-            visualEditorWrapper?.appendChild(replaceButton);
             return;
         }
 
```

Comment:
Since we are removing the replace button from here, do we see a replace button anywhere?

Reply:
I didn't quite get you here. On a side note, I moved `visualEditorWrapper?.appendChild(replaceButton);` inside `generateReplaceAssetButton` itself. I updated the code a little later.